src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  useContext, useRef, useState,
+} from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodosContext } from '../../contexts/TodosProvider';
+import { TodoAction } from '../../types/TodoAction';
+
+type Props = {
```

Comment:
```suggestion
interface Props  {
```
use interface prefer